CHANGELOG.md

```diff
@@ -19,5 +19,7 @@ git logs & PR history.
 
 - concept of `manual_test` and `manual_step`
 
```

Comment:
Please put your changes below Unreleased with an Added section.  So it should look like:

# [Unreleased](https://github.com/puppetlabs/beaker/compare/3.26.0...master)

### Added

- fine-grained, structured timing for tests